app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.kt

```diff
@@ -113,6 +113,15 @@ class ZimFileSelectFragment : BaseFragment() {
     }
   }
 
+  override fun setUserVisibleHint(isVisibleToUser: Boolean) {
```

Comment:
this method is deprecated, I would avoid using it

Reply:
@macgills I was unable to find another method which is called when the fragment made visible or hidden. I tried using onHiddenChanged(), but that did not work. 

I tried using `userVisibleHint` too inside onPause(), that didn't work.

Reply:
You could go to the activity and add an onPageChangeListener to the viewPager

Reply:
@macgills `SimplePageChangeListener` already exists. I tried making a function in `ZimFileSelectFragment` and calling it in the listener, but I wasn't able to make it work.

Reply:
It is possible, push your attempt and I can help guide you

Reply:
@macgills I pushed some changes that I made but didn't work. Please guide me. 

I'll squash all extra commits once done.

Reply:
The same thing I've done in ZimManageAcitivty, I placed the same code in the onPageScrollStateChanged of SimplePageChangeListener. That also didn't work.